src/poly/coeffs.rs

```diff
@@ -164,7 +165,7 @@ where
     #[instrument(skip_all)]
     pub fn evaluate_at_extension<EF: ExtensionField<F>>(&self, point: &MultilinearPoint<EF>) -> EF {
         assert_eq!(self.num_variables, point.num_variables());
-        eval_extension(&self.coeffs, &point.0, EF::ONE)
+        eval_extension_para(&self.coeffs, &point.0)
```

Comment:
Maybe this renaming is better to keep aligned with the rayon namings?

```suggestion
        eval_extension_par(&self.coeffs, &point.0)
```